src/magicbox/SuggestionsManager.ts

```diff
@@ -206,14 +222,28 @@ export class SuggestionsManager {
     $$(this.root).trigger(OmniboxEvents.querySuggestRendered);
   }
 
+  private afterMoving(selected) {
```

Comment:
This method should be converted to a public getter that returns the active preview. e.g. `public get selectedPreviewElement`.

As for the two types, `Suggestion` and `HTMLElement`, do we do any processing when we get the html element? If not, I would either A) return a "null suggestion" object, or B) return `null`.

Reply:
Exactly what I did when refactoring to return `void` in the `move` function 😄